unintended_ml_bias/new_madlibber/enums.py

```diff
@@ -0,0 +1,9 @@
+def enum(**enums):
+  values = set([value for value in enums.values()])
+  enums['exists'] = values.__contains__
+  return type('Enum', (), enums)
+
+Toxicity = enum(TOXIC='toxic', NONTOXIC= 'non-toxic')
+WordType = enum(NAME='name', ADJECTIVE='adjective', IDENTITY='identity', OCCUPATION='occupation', VERB='verb')
+WordConnotation = enum(POSITIVE='positive', NEGATIVE='negative', NEUTRAL='neutral')
+WordGender = enum(MALE='m', FEMALE='f', NEITHER='n')
```

Comment:
Could these things be inferred from the templates?

Reply:
If we assume that the values in the files will always be correct, we can absolutely infer these instead. Currently, the existence of these enums is for sanity checking the files (in which case we would not want to infer from the files). If we don't need to do this, I think we will not need these enums at all.

Reply:
Yes, I think it's fine to use the template as the source of truth and expect the words categories to match what the template expects (and throw an error if there are missing word categories).

Reply:
Ohh okay, I will work on this, thanks!